python/cucim/src/cucim/core/operations/color/tests/test_color_jitter.py

```diff
@@ -0,0 +1,66 @@
+import cupy
+import numpy as np
+import pytest
+import skimage.data
+from PIL import Image
+
+import cucim.core.operations.color as ccl
+
+
+def get_image_array():
+    img = skimage.data.astronaut()
+    arr = np.asarray(img)
+    arr = np.transpose(arr)
+    return arr
+
+
+def verify_result(output, input):
+    diff_im = output - input
+    diff_total_value = np.abs(np.sum(diff_im))
+    assert diff_total_value >= 0
+
+
+def test_color_jitter_param():
```

Comment:
Since this seems focused on testing error cases, maybe we can incorporate that into the test function name for clarity 

```suggestion
def test_color_jitter_bad_params():
```